modules/plugin/geotiff/pom.xml

```diff
@@ -119,7 +119,12 @@
       <groupId>it.geosolutions.imageio-ext</groupId>
   	  <artifactId>imageio-ext-tiff</artifactId>
       <!-- The version number is specified in the parent POM. -->      
-    </dependency>   
+    </dependency>
+    <dependency>
```

Comment:
Is this dependency big? Would it make sense to make it optional here in GeoTools, maybe not everyone needs to read COGs. Could be part of the standard deps in GeoServer, or maybe even better a plugin (if memory serves me right, S3 deps are really big, though maybe this cog-reader artifact does not depend on them directly).

Reply:
Actually, the S3-range-reader (and related aws deps) is not a dependency of the reader